client/src/App.js

```diff
@@ -1,11 +1,74 @@
 import "./App.css";
+import DeleteVideo from "./component/DeleteVideo";
+import Search from "./component/Search";
+import VideoRatings from "./component/VideoRatings";
+import Videos from "./component/Videos";
+import VideoTitle from "./component/VideoTitle";
+import React, { useState } from "react";
+
+import data from "./data/exampleresponse.json";
+import AddVid from "./component/AddVid";
 
 function App() {
+  const [searchValue, setSearchValue] = useState("");
+  const [videos, setVideos] = useState(data);
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+  const [showVideoForm, setShowVideoForm] = useState(false);
+
+  // Search functionality
+  const filteredTitle = videos.filter((video) =>
+    video.title.toLowerCase().includes(searchValue.toLowerCase())
+  );
+
+  // Handle submit function when a new video is added
+  const addNewVidSubmit = (e) => {
+    e.preventDefault();
+    let newVideo = { title, url, rating: 0 };
+    setVideos(() => [...videos, newVideo]);
+    setTitle({ title: "" });
+    setUrl({ url: "" });
```

Comment:
This doens't clear the html form, so when you press the submit button again, it crashes